packages/flex-plugin-scripts/src/config/devServer/pluginServer.ts

```diff
@@ -153,6 +154,13 @@ export const _startServer = (plugins: StartServerPlugins, config: StartServerCon
           return remotePlugins;
         }
 
+        // Check that all remote plugins inputted are valid
+        for (const plugin of plugins.remote) {
```

Comment:
Update slightly so `remotePluginNotFound` accepts a list of plugins not found and prints them all, something like:


```js
const notFoundPlugins = plugins.filter(plugin => remotePlugins.find(r => r.name === plugin));
remotePluginNotFound(notFoundPlugins, remotePlugins);

// Now show that all of these plugins are not found
```